_includes/contact-info.html

```diff
@@ -25,6 +25,11 @@
 List subteams within the larger editorial team
 {% endcomment %}
 <p>
+{% for status in member.status %}
```

Comment:
This shouldn't need to be a for loop since everyone only has 1 `status`

Reply:
This is why we pay you the big bucks.

Reply:
@walshbr actually we don't pay him at all... ;P

Reply:
This should now be working @mdlincoln . Thanks.